docs/dev/disk-image-install.md

```diff
@@ -0,0 +1,22 @@
+# Assisted Installation With a Disk Image
+
+You can use the Assisted Installer to install to install directory to the existing root filesystem of a disk image.
+This should be done for platforms that cannot take the discovery ISO as boot media, and also cannot use iPXE.
+
+## Available Disk Images
+
+You can download disk images for use with this process at https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos
+An example for installing OCP 4.17 with libvirt would be https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.17/4.17.2/rhcos-4.17.2-x86_64-qemu.x86_64.qcow2.gz
+
+## Discovery and Installation
+
+You must write the Assisted Installer discovery ignition to the disk on first boot in order to install in this way.
+You can do this using whatever metadata service is available for your target platform.
```

Comment:
Because I'm not familiar with this 😅 but it would be helpful to have an example of a service or how to do this?

Reply:
But if it's common knowledge that doesn't need to be documented, then feel free to ignore my comment

Reply:
I don't know.

I was thinking of adding an example of how I was doing this for testing, but I didn't feel like that would really be helpful for anyone who wanted to do this on another platform.

I don't think it makes sense to go into everything you and Riccardo are doing now for CAPI around this either here so I left it pretty vague. I suppose I could link https://coreos.github.io/ignition/supported-platforms/#supported-platforms which talks a bit about how ignition is pulled for all those platforms?

Reply:
For example I was doing this:

>[QEMU](https://www.qemu.org/) (qemu) - Ignition will read its configuration from the ‘opt/com.coreos/config’ key on the QEMU Firmware Configuration Device (available in QEMU 2.4.0 and higher).

But I don't think how to do that is particularly helpful here.

Reply:
Ah I can see why you didn't document a concrete example. The link would definitely be helpful for anyone who wants to try this out on one of the platforms!